src/Command/CommandHelper.php

```diff
@@ -303,7 +303,7 @@ public static function begin(
 				})($bootstrapFile);
 			} catch (\Throwable $e) {
 				$errorOutput->writeln($e->getMessage());
```

Comment:
IMO, in cases where we throw `InceptionNotSuccessfulException` with a root-cause exception provided, we should not print our the message manually but let symfony-console do its magic.

(this would apply to a few more places within this file)

Reply:
We could print out the stack trace in this place manually. The same way Symfony Console does. Can you investigate what function should we call to do that?

Reply:
there is a `Symfony\Component\Console\Application::renderException()`.. I will have a look